pkg/kosmosctl/install/install.go

```diff
@@ -0,0 +1,205 @@
+package install
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	extensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	appsclient "k8s.io/client-go/kubernetes/typed/apps/v1"
+	coreclient "k8s.io/client-go/kubernetes/typed/core/v1"
+	rbacclient "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	ctlutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/manifest"
+	"github.com/kosmos.io/kosmos/pkg/kosmosctl/util"
+	"github.com/kosmos.io/kosmos/pkg/version"
+)
+
+const (
+	clusterlinkSystem         = "clusterlink-system"
```

Comment:
should it be kosmos-system

Reply:
well~considering the current stability，I will modify this after other modules are installed and accessed @duanmengkk